tests/functional/aws-node-sdk/test/versioning/objectACL.js

```diff
@@ -14,14 +14,16 @@ const counter = 100;
 let bucket;
 const key = '/';
 const invalidId = 'invalidId';
-const nonExistingId = '3939393939393939393936493939393939393939756e6437';
+const nonExistingId = process.env.AWS_ON_AIR ?
+    'MhhyTHhmZ4cxSi4Y9SMe5P7UJAz7HLJ9' :
```

Comment:
I added a comment. It's to have the correct format when running against AWS, otherwise tests for non-existing ids will get InvalidArgument instead of NoSuchKey.